app/src/main/java/xyz/whereuat/whereuat/ui/views/ContactCard.java

```diff
@@ -54,10 +58,35 @@ public void run() {
                         // Select the phone number of the contact with this id.
                         String contact_id = ContactCard.this.getTag().toString();
                         Cursor phone = ContactUtils.buildSelectContactByIdCommand(context,
-                                contact_id, new String[] {ContactEntry.COLUMN_PHONE}).call();
+                                contact_id, new String[] {ContactEntry.COLUMN_PHONE,
+                                        ContactEntry.COLUMN_REQUESTS}).call();
 
                         // Get the contact's phone number from the cursor and send them an @request.
                         if (phone.moveToFirst()) {
+                            ContentValues cv = new ContentValues();
+                            int col_i = phone.getColumnIndexOrThrow(ContactEntry.COLUMN_REQUESTS);
+                            int num_requests = -1;
+                            try {
+                                num_requests = phone.getInt(col_i);
+                            } catch (Exception e) {
+                                e.printStackTrace();
+                            }
+                            final int curr_num_requests = num_requests + 1;
```

Comment:
this seems like a lot of floofypoof just to get `curr_num_requests`. can you think of a way to consolidate? at least put the col_i and `num_requests` stuff into the try-catch.